packages/create-cloudflare/src/pages.ts

```diff
@@ -205,7 +210,9 @@ const createProject = async (ctx: PagesGeneratorContext) => {
 					cwd: ctx.project.path,
 					env: { CLOUDFLARE_ACCOUNT_ID },
 					startText: "Creating Pages project",
-					doneText: `${brandColor("created")} ${dim(`via \`${cmd.trim()}\``)}`,
+					doneText: `${brandColor("created")} ${dim(
+						`via \`${cmd.join(" ").trim()}\``
```

Comment:
(Related to my more general comment https://github.com/cloudflare/workers-sdk/pull/4445#issuecomment-1814872733)

This is an example of where we still end up using .join(" ") – if command has an values with spaces, this will stringify incorrectly without quotes